test/api/application/fixtures.go

```diff
@@ -0,0 +1,70 @@
+package application
+
+import (
+	"encoding/json"
+	"fmt"
+	"testing"
+
+	"github.com/konveyor/tackle2-hub/api"
+	"github.com/konveyor/tackle2-hub/test/api/testclient"
+)
+
+// Hub REST API/addon Client
+// Setup Hub API client
+var Client, err = testclient.NewHubClient()
+
+//if err != nil {
+//	t.Fatalf("Unable connect to Hub API: %v", err.Error())
+//}
+
+//
+// Set of valid Application resources for tests and reuse.
+// Invalid application for negative tests are expected to be defined within the test methods, not here.
+var SampleApplications = []*api.Application{
+	{
+		Name:        "Pathfinder",
+		Description: "Tackle Pathfinder application.",
+		Repository: &api.Repository{
+			Kind:   "git",
+			URL:    "https://github.com/konveyor/tackle-pathfinder.git",
+			Branch: "1.2.0",
+		},
+	},
+	{
+		Name: "Minimal application",
+	},
+}
+
+//
+// Creates a copy of SampleApplication for given test (copy is there to avoid tests inflence each other using the same object ref).
+func Samples() (applications []*api.Application) {
+	raw, err := json.Marshal(SampleApplications)
+	if err != nil {
+		fmt.Print("ERROR cloning samples")
+	}
+	json.Unmarshal(raw, &applications)
+	return
+}
+
+//type TestCase struct {
+//	Test        testclient.TestCase
+//	Application *api.Application
+//}
+
+//
+// Create an Application (and stop tests on failure).
+func Create(t *testing.T, application *api.Application) {
```

Comment:
I did, partially because this becomes handy for the same resource testing (e.g.Create() before Update test and Delete() after the test). I'm likely not implementing other mathods than Create&Delete unless see a use-case.